spicedb.go

```diff
@@ -78,12 +79,16 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 		return nil, err
 	}
 
+	c := &spiceDBContainer{Container: container, secretKey: cfg.SecretKey, model: cfg.Model}
+
 	endpoint, err := container.Endpoint(ctx, "")
 	if err != nil {
-		container.Terminate(ctx)
-		return nil, err
+		return c, err
```

Comment:
lint was complaining because the error was not handled, so I discovered this here